### PR TITLE
feat(Replit): Add new logo, and add support for new areas

### PR DESCRIPTION
**Pull request by totally-google-drive**
## Description 
Updated to the new Replit logo, and added support for the new(ish) areas (Bounties, Learn, Themes, etc..)
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
image
image
image
image

</details>

### CODE REVIEW

File: websites/R/Replit/presence.ts [hunk 1]
```diff
@@ -1,89 +1,110 @@
...
+        presenceData.details = "Viewing home page";
+        }
+    else if (pathname.startsWith("/themes")) {
+        presenceData.details = "Browsing themes";
+        }
+        else if (pathname.startsWith("/theme")) {
+            const pathArray = pathname.split("/");
+            if (pathArray.length > 2 && pathArray[pathArray.length - 1].length > 0) {
+              const themeName = pathArray[pathArray.length - 1].replace(/-/g, " ");
+              presenceData.details = `Admiring a theme`;
```
Comment (theusaf) [inline websites/R/Replit/presence.ts:79]:
```suggestion
              presenceData.details = "Admiring a theme";
```

File: websites/R/Replit/metadata.json [hunk 2]
```diff
@@ -20,13 +20,13 @@
 	},
 	"url": "replit.com",
 	"version": "1.0.13",
-	"logo": "https://i.imgur.com/jL7ohN5.png",
-	"thumbnail": "https://i.imgur.com/4AwIxoc.png",
+	"logo": "https://i.imgur.com/ge3dweY.png",
```
Comment (theusaf) [inline websites/R/Replit/metadata.json:23]: Logos must be 512x512px

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by theusaf** [comment]
* It looks like you uploaded the javascript file rather than the typescript file.
* Resolve CI issues

**Comment by Bas950** [comment]
Closing for inactivity

**Pull request closed without merging**